apache/no_default_vhost.sls

```diff
@@ -5,12 +5,13 @@
 include:
   - apache
 
-apache_no-default-vhost:
-  file.absent:
-    - name: {{ apache.vhostdir }}/000-default.conf
+a2dissite 000-default.conf:
+  cmd:
+    - run
```

Comment:
This mixes two styles, namely `cmd.run` and `cmd: - run`. It would be nice to stick to `cmd.run:` everywhere.